src/redux/actions/projects/createProject.js

```diff
@@ -24,6 +31,8 @@ const createProject = (
     type: PROJECTS_CREATE,
     payload: { project: newProject },
   });
+
+  dispatch(saveProject(newProjectUuid));
```

Comment:
It looks like the action `createProject` is just a wrapper around `saveProject`. In this case, is it worth having `createProject` at all? For creating project we can just call `saveProject`. Also, you are missing an action type for when there is an error with project creation

Reply:
`createProject` creates a new project and saves the project into the store. `saveProject` saves the project into Dynamo. It is refactored into its own action because there are multiple places where we want to save the project in Dynamo.